app/assets/stylesheets/application.css.scss

```diff
@@ -46,6 +64,26 @@ body {
   }
 }
 
+.avatar-selection{
+  a {
+    color: white;
+    font-size: 3.5em;
+    font-family: 'Tangerine', cursive;
+    line-height: 1.25em;
+    &:hover{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space